tests/datavolume_test.go

```diff
@@ -535,7 +542,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 					Message: "Import Complete",
 					Reason:  "Completed",
 				}}),
-			table.Entry("[rfe_id:1115][crit:high][test_id:1379]succeed creating import dv with given valid qcow2 url (https) should require scratchspace", dataVolumeTestArguments{
+			table.PEntry("[rfe_id:1115][crit:high][test_id:1379]succeed creating import dv with given valid qcow2 url (https) should require scratchspace", dataVolumeTestArguments{
```

Comment:
I think you can leave this as it, maybe remove the 'should require scratchspace' part of the test label. We never actually check that scratch space is created anyway. We just make sure the import completes successfully.

Reply:
And when I say leave as is, I mean as it was originally by running the test. It should not fail.

Reply:
the test works and I changed the test description